### PR TITLE
Cisco: Qos-sai: Change the tests to use only downlink ports. 

### PR DESCRIPTION
**Pull request by rraghav-cisco**
Porting 10381 to master, along with some more mods to fix precommit errors:
Addressing the test fails when they use uplink ports. Forcing the script to use downlink ports only.

### CODE REVIEW

**Comment by rraghav-cisco** [comment]
@XuChen-MSFT : were you able to verify this PR ? Thanks.

File: tests/qos/test_qos_sai.py [hunk 1]
```diff
@@ -1118,7 +1118,7 @@ def testQosSaiLossyQueue(
     def testQosSaiLossyQueueVoq(
         self, LossyVoq, ptfhost, dutTestParams, dutConfig, dutQosConfig,
             ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts,
-            skip_src_dst_different_asic
+            skip_src_dst_different_asic,  separated_dscp_to_tc_map_on_uplink
```
Comment (XuChen-MSFT) [inline tests/qos/test_qos_sai.py:1121]: @rraghav-cisco 
conflict on 202305 branch,
can you file a separate PR to doble commit for 202305 branch?

Comment (rraghav-cisco) [reply tests/qos/test_qos_sai.py:1121]: @XuChen-MSFT : 3 PRs are open:
202205: https://github.com/sonic-net/sonic-mgmt/pull/10381
master: https://github.com/sonic-net/sonic-mgmt/pull/10896
202305: https://github.com/sonic-net/sonic-mgmt/pull/10898

**Comment by rraghav-cisco** [comment]
@XuChen-MSFT : All 3 PRs are updated based on the findings from the debug session. Thanks for your time.

**Review by XuChen-MSFT — APPROVED** [review]
LGTM, pass test on local cisco 8000 platform

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**